vhdbuilder/packer/install-dependencies.sh

```diff
@@ -42,6 +42,9 @@ chmod 666 /opt/certs
 systemctlEnableAndStart update_certs.path || exit 1
 systemctlEnableAndStart update_certs.timer || exit 1
 
+systemctlEnableAndStart ci-syslog-watcher.path || exit 1
```

Comment:
just confirming, this works on 18.04, 20.04, 22.04, 18.04 FIPS, Mariner 1 and 2? 

I don't see a reason it wouldn't for Ubuntu, but forget if we already discussed this for Mariner?

Reply:
@alexeldeib 

i have test mariner 1, 2, ubuntu 22.04 and ubuntu 18.04. Do I need to test it on 18.04 FIPS and 20.04 as well?

Reply:
nah that's fine, if it works on 18.04 and 22.04 I doubt it broke on 20.04 :) just emphasizing this is hitting all versions.